docs/content/action/output.md

```diff
@@ -0,0 +1,31 @@
+---
+navtitle: Output
+sort: 3
+---
+
+# Action output
+
+Action output is the result returned by Nikita's actions. It is defined by the [action handler](/current/action/handler#return) enriched with the default values.
```

Comment:
It is the value return by an action handler eventually modified by plugins and wrapped inside a promise

Reply:
> by plugins
Where we describe what is Nikita plugin?